src/screens/AddNewItemScreen/style.ts

```diff
@@ -5,19 +5,24 @@ type AddNewItemScreenStyleType = {
   container: ViewStyle;
   button: ViewStyle;
   buttonDis: ViewStyle;
-  input: ViewStyle;
   text: TextStyle;
   textDis: TextStyle;
+  inputContainer: ViewStyle;
 };
 
 export const style = StyleSheet.create<AddNewItemScreenStyleType>({
   container: {
     padding: 10,
     flexDirection: 'column',
+    alignItems: 'center',
     justifyContent: 'space-between',
     flex: 1,
   },
+  inputContainer: {
+    width: '100%',
+  },
   button: {
+    width: '100%',
```

Comment:
your `button` and `buttonDis` style almost the same

I've already told you what we should do in this case (also you can see that in your current case you have to duplicate style everywhere and in the future it might be hard not to miss any case)